### PR TITLE
Add countries to epiv1 datapivot export

### PR DESCRIPTION
**Pull request by caseyhans**
Add countries to the epiv1 datapivots.

### CODE REVIEW

File: hawc/apps/epi/exports.py [hunk 1]
```diff
@@ -593,6 +595,7 @@ def build_df(self) -> pd.DataFrame:
                 "sp-age_profile": "study population age profile",
                 "sp-source": "study population source",
                 "sp-design": "design",
+                "sp-countries": "countries",
```
Comment (shapiromatron) [inline hawc/apps/epi/exports.py:598]: change to `str_m2m()`? this would just show one if there were more than one wouldn't it?

Comment (caseyhans) [reply hawc/apps/epi/exports.py:598]: this is already done in the study population export! all we had to do was include it in the datapivot study population module
https://github.com/shapiromatron/hawc/blob/1be48496e5be3778526da4f927f03c28bd50a0a8/hawc/apps/epi/exports.py#L60

here's how it looks in a datapivot:
image

Comment (shapiromatron) [reply hawc/apps/epi/exports.py:598]: oh, nevermind! it's already there, abstracted away. https://github.com/shapiromatron/hawc/blob/1be48496e5be3778526da4f927f03c28bd50a0a8/hawc/apps/epi/exports.py#L60

Comment (shapiromatron) [reply hawc/apps/epi/exports.py:598]: haha you beat me too it!!

**Review by munnsmunns — APPROVED** [review]
Looks good Casey!

**Review by shapiromatron — APPROVED** [review]

**Pull request merged**